tests/README.md

```diff
@@ -179,28 +179,36 @@ For example `EC2` data cases will be located under `aws_ec2_test_cases.py`.
 
 - Define data manually in AWS Cloud and define / get property for resource unique identification
 - Create test case data in `data` folder, for example in file `aws_logging_test_cases.py`
+
 ```python
 cis_aws_log_3_1_pass = EksAwsServiceCase(
     rule_tag=CIS_3_1,
     case_identifier="cloudtrail-704479110758", # resource unique identifier
     expected=RULE_PASS_STATUS,
 )
 ```
+
 - Update the test cases dictionary or create a new one if it didn't exist, for example:
+
 ```python
 cis_aws_log_3_1 = {
     "3.1 Ensure CloudTrail is enabled in all regions expect: passed": cis_aws_log_3_1_pass,
 }
 ```
+
 - Finally, reference created dictionary in the group of all test cases, for example
+
 ```python
 cis_aws_log_cases = {
     **cis_aws_log_3_1,
     ...
 ```
-- If just adding a new test case to existing test suite, no additional steps are required, the case will be added automatically
+
+- If just adding a new test case to existing test suite, no additional steps are required, the case will be added
+  automatically
```

Comment:
revert

Reply:
this is automatically formatted by our pre-commit hook to enforce line length (it won't affect the actual markdown output)